chaste_codegen/__init__.py

```diff
@@ -6,6 +6,8 @@
 from cellmlmanip.transpiler import Transpiler
 
 from chaste_codegen.be_model import BeModel  # noqa
+from chaste_codegen.rl_model import RlModel  # noqa
```

Comment:
I'd generally prefer more explicit names even if they are a bit longer, so `BackwardEulerModel` and `RushLarsenModel`.